File: solutions/_1980_find_unique_binary_string/solution.cpp
#include <vector>
#include <set>
#include <string>
using namespace std;

class Solution
{
public:
    string findDifferentBinaryString(vector<string> &nums)
    {
        int n = nums.size();
        set<string> s(nums.begin(), nums.end());
        for (int i = 0; i < 1 << n; i++)
        {
            string t;
            for (int j = 0; j < n; j++)
                t += (i >> j & 1) + '0';
            if (!s.count(t))
                return t;
        }
        return "";
    }
};

/*
https://leetcode.com/problems/find-unique-binary-string/
*/

File: solutions/_1980_find_unique_binary_string/solution.java

package _1980_find_unique_binary_string;

import java.util.Set;

class Solution {
    public String findDifferentBinaryString(String[] nums) {
        int n = nums.length;
        Set<String> set = Set.of(nums);
        for (int i = 0; i < 1 << n; i++) {
            String s = Integer.toBinaryString(i);
            while (s.length() < n) {
                s = "0" + s;
            }
            if (!set.contains(s)) {
                return s;
            }
        }
        return "";
    }
}

/*
 * https://leetcode.com/problems/find-unique-binary-string/
 */